extensions/graphite-emitter/README.md

```diff
@@ -0,0 +1,69 @@
+## introduction
+
+This extension emits some selected druid metrics to a graphite carbon server.
```

Comment:
We can remove the word "selected" as we now support sending all metrics as well.